dev-tools/validation/check-authz-route-matrix.py

```
@@ -114,6 +329,1079 @@ def validate_auth_path_vocabulary(document: dict[str, Any], errors: list[str]) -
+                    f"{label}.generations.{generation} must be {expected_value!r}"
+                )
+        if entry.get("privileged_control") not in PRIVILEGED_CONTROL_VALUES:
+            errors.append(
+                f"{label}.privileged_control must be one of "
+                f"{sorted(PRIVILEGED_CONTROL_VALUES)}"
+            )
+
+
+def validate_membership_policy(
+    document: dict[str, Any],
+    routes: list[Any],
```

Ruff (0.16.0)

[warning] Unused function argument: `routes`

(ARG001)